components/automate-chef-io/content/docs/reports.md

```diff
@@ -75,9 +74,9 @@ Please note the only supported filters for deep filtering are:
 
 ### Waivers
 
-A node's waived status appears if applicable in displays where a node's status appears in Chef Automate. 
+A node's waived status appears if applicable in displays where a node's status appears in Chef Automate.
 The Compliance Reports overview displays the node count and history of waived nodes, and the count and history of waived controls.
-_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively. 
+_Nodes_ and _Profiles_ views include _Waived Nodes_ and _Waived Profiles_ status filters respectively.
```

Comment:
This section is done as a stop-gap documentation measure because waivers were originally added without documentation. I would greatly like to revise and add to this further.

Reply:
It's possible that the product has changed enough that we need to review the need for this page to exist.